.travis.yml

```diff
@@ -1,11 +1,27 @@
 language: ruby
-sudo: false
+sudo: required
+services: docker
+env:
+  matrix:
+    - INSTANCE=default-centos-7
+    - INSTANCE=default-ubuntu-1404
+    - INSTANCE=default-debian-8
```

Comment:
uhhhhh maybe debian sid?? :) If there's one for that we should mark it allowed to fail